init.lua

```diff
@@ -78,25 +113,29 @@ function filter.mute(name, duration)
 		minetest.set_player_privs(name, privs)
 	end
 
-	minetest.chat_send_player(name, "Watch your language! You have been temporarily muted")
+	minetest.chat_send_player(name, "You have been temporarily muted for abusing the chat.")
```

Comment:
changes text, so, separate commit?

Reply:
That should ideally be a separate commit, but I've already added a few more commits after this. Is it possible to revert this now?